.github/workflows/build.yml

```diff
@@ -8,21 +8,21 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
-    container: ghcr.io/zephyrproject-rtos/ci:v0.26.2 
-    env:
-      CMAKE_PREFIX_PATH: /opt/toolchains
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           path: example-application
 
-      - name: Initialize
-        working-directory: example-application
-        run: |
-          west init -l .
-          west update -o=--depth=1 -n
+      - name: Setup Zephyr project
+        uses: fabiobaltieri/zephyr-setup@main
```

Comment:
Nice :)

Please do let me know if you actually plan to pursue this. We can consider moving it to the zephyrproject-rtos org.

p.s. any plan to support Windows?

Reply:
Yeah I was thinking about proposing to have it moved under the org. Unsure about Windows support, I'll have to play with it a bit and see what's involved.

Reply:
Added windows support, wasn't as bad as I thought it would after all.